lib/pages/people-page.js

```diff
@@ -94,19 +94,17 @@ export default class PeoplePage extends BaseContainer {
 	 * @param {bool} useSince Whether to include the "SINCE" line in the pattern match (for Followers, not Viewers)
```

Comment:
I think this should be removed as not needed

Reply:
Done! Thanks!